draft-ietf-ccwg-rfc5033bis.md

```diff
@@ -264,6 +260,32 @@ deployed congestion control algorithms that cannot be changed by IETF or IRTF re
 are invited to publish as an Informational RFC via the Independent Stream Editor
 (ISE).
 
+# Specifying Algorithms for use in Controlled Environments
+
+Algorithms can be designed for general Internet deployment or for use in controlled environments.
+A controlled environment can form all or part of the Internet path
+or might carry network traffic that is isolated from the end-to-end path.
```

Comment:
I confess that I don't know what this sentence means: 
  "might carry network traffic that is isolated from the end-to-end path"
Sorry if I'm being dense. :-)

Reply:
Not dense, the relocated text didn't make sense. New words suggested.

Reply:
Thanks! Looks good to me now at f1d5dd25a0ba4e7831bef9958f00e60083e183e1